app/tests/integration/nimbus/test_mobile_targeting.py

```diff
@@ -78,6 +78,7 @@ def test_check_mobile_targeting(
             "days_since_update": 1,
             "days_since_install": 1,
             "isFirstRun": "true",
+            "is_first_run": True,

```

Comment:
Why is there both `isFirstRun` and `is_first_run`?

Reply:
Both exist within the Fenix repo, `isFirstRun` is a string and only there for legacy purposes, they've moved to `is_first_run` as a Boolean going forward.

Reply:
Ah interesting 👍